bin/verilator

```diff
@@ -968,6 +969,10 @@ stylistic and not enabled by default.
 If the design is not to be completely Verilated see also the --bbox-sys and
 --bbox-unsup options.
 
+=item --max-num-width I<value>
+
+Overwrite the maximum number literal width of 64K with the given value.
```

Comment:
Set the maximum number literal width (e.g. in 1024'd22 this it the 1024). Defaults to 64K.